curriculum/challenges/english/01-responsive-web-design/basic-css/use-clockwise-notation-to-specify-the-margin-of-an-element.english.md

```diff
@@ -16,7 +16,7 @@ These four values work like a clock: top, right, bottom, left, and will produce
 
 ## Instructions
 <section id='instructions'>
-Use <code>Clockwise Notation</code> to give the element with the <code>blue-box</code> class a margin of <code>40px</code> on its top and left side, but only <code>20px</code> on its bottom and right side.
+Use <dfn>Clockwise Notation</dfn> to give the element with the <code>blue-box</code> class a margin of <code>40px</code> on its top and left side, but only <code>20px</code> on its bottom and right side.
```

Comment:
In order to use dfn tags here, `Clockwise Notation` would have to be defined, but it is not, so remove them.